packages/@aws-cdk/aws-gamelift/lib/script.ts

```diff
@@ -0,0 +1,237 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as s3_assets from '@aws-cdk/aws-s3-assets';
+import * as cdk from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { Content } from './content';
+import { CfnScript } from './gamelift.generated';
+
+/**
+ * Represents a GameLift realtime server script.
```

Comment:
I think the explanation from the "About" page communicates, in a short paragraph, what a script is:

```suggestion
 * Your configuration and custom game logic for use with Realtime Servers.
 * Realtime Servers are provided by GameLift to use instead of a custom-built
 * game server. You configure Realtime Servers for your game clients by
 * creating a script using JavaScript, and add custom game logic as
 * appropriate to host game sessions for your players. You upload the Realtime
 * script to the GameLift service in the Regions where you plan to set up 
 * fleets.
 *
 * @see https://docs.aws.amazon.com/gamelift/latest/developerguide/realtime-script-uploading.html
```

I suggest you also copy the one for builds to the `Build` class.